### PR TITLE
Update privacy-personal-data.md

### PR DESCRIPTION
**Pull request by ghnu**
jaimeo

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/managed-desktop/service-description/privacy-personal-data.md [hunk 1]
```diff
@@ -42,6 +42,10 @@ Microsoft Managed Desktop stores its data in one or more of the following Micros
 
 Microsoft Managed Desktop stores its data in the United States. Personal data is retained by Microsoft Managed Desktop for a maximum of 30 days.
 
+### Staff location
+
+The MMD Operations and MMD Security Operations teams are located in the United States and India.
```
Comment (JohanFreelancer9) [inline microsoft-365/managed-desktop/service-description/privacy-personal-data.md:47]:
```suggestion
The Microsoft Managed Desktop Operations and Microsoft Managed Desktop Security Operations teams are located in the United States and India.
```

Comment (JohanFreelancer9) [reply microsoft-365/managed-desktop/service-description/privacy-personal-data.md:47]: "Microsoft Managed Desktop" has not been abbreviated anywhere else in this topic, so this suggestion is purely for consistency.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

File: microsoft-365/managed-desktop/service-description/privacy-personal-data.md (hunk 1 above)
Comment (jaimeo) [reply microsoft-365/managed-desktop/service-description/privacy-personal-data.md:47]: @JohanFreelancer9 correct: Microsoft Managed Desktop must _never_ be abbreviated.

**Pull request merged**